src/strategies/bearerTokenStrategy.ts

```diff
@@ -28,7 +29,10 @@ const bearerTokenStrategy = (
     done: (error: any, user?: any, options?: IVerifyOptions | string) => void
   ) => {
     const path = req.route.path;
-    const sessionStorage: ISessionStorage = SESSION_STORAGE;
+    const sessionStorage: ISessionStorage = new RedisSessionStorage(
```

Comment:
that would be better to pass the sessionstorage as a param, see:

https://github.com/AgID/agid-forms-backend/blob/master/src/strategies/bearer_token.ts

Reply:
Fixed in 3e0d913